tests/common/lib/bin.bash

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+# Helpers to run templated bin tests
```

Comment:
Something rubs me the wrong way calling these "helpers" since they are very specific tests. It's not like they can be shared across different tests, right?

Reply:
Aren't these templates since they are used to render the templated tests? :smile:

Reply:
Not really, these are plain test functions that the templated tests are using.
I'll see if I can rework the generator to add functions.

Reply:
I see, they look "complete", as in what else does the generator add to them? I should probably generate the tests and just see for myself. :smile:

Reply:
Let's not waste more time bikeshedding over this though haha.